twake/frontend/src/app/scenes/App.tsx

```diff
@@ -22,9 +22,5 @@ export default () => {
     (Globals as any).store_publicAccess_get_data = WindowService.allGetParameter();
   }
 
-  return (
-    <BadDevice force={publicAccess}>
-      <ClientPage />
-    </BadDevice>
-  );
+  return <ClientPage />;
```

Comment:
Can you put again the "BadDevice" here?
We need it for old browsers too, so please keep it and just adapt the condition if you can :)
We don't support: IE and Opera mobile

Reply:
Now minimum screen width is set to 300px